src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+  display: block;
+}
```

Comment:
1. Move font properties to `html` or `:root`.
2. `nav` is `display: block;` by default